apps/cms/models.py

```diff
@@ -116,3 +120,57 @@ def __str__(self):
 
 class NavigationMenuItem(Orderable, MenuItem):
     parent = ParentalKey('meinberlin_cms.NavigationMenu', related_name='items')
+
+
+class EmailFormField(AbstractFormField):
+    page = ParentalKey('EmailFormPage', related_name='form_fields')
+
+
+class EmailFormPage(AbstractEmailForm):
+    intro = fields.RichTextField(
+        help_text='Introduction text shown above the form'
+    )
+    thank_you = fields.RichTextField(
+        help_text='Text shown after form submission',
+    )
+    email_content = models.CharField(
+        max_length=200,
+        help_text='Email content message',
+    )
+    attach_as = models.CharField(
+        max_length=3,
+        choices=(
+            ('csv', 'CSV Document'),
+        ),
+        default='csv',
+        help_text='Form results are send in this document format',
+    )
```

Comment:
I had initially an additional option to put the responses in the mail, but removed it as it would require extending the email template system.

I still decided to keep this option to have the CSV generation more explicit.